advocacy_docs/repos/migration/index.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Migrating from Repos 1.0 to 2.0
+navTitle: Migrating to Repos 2.0
+description: How to migrate from the old EDB Repositories to the new EDB Repositories
+deepToC: true
+---
+
+There are two steps to this process and we recommend that you do them in order. The first step is to remove the old Repos 1.0 repositories from your system. The second step is to add the new Repos 2.0 repositories to your system. It is possible to perform the steps in the reverse order, but you will have to ensure that no proceses are run during the migration that may attempt to update or install packages. 
+
+## Remove Repos 1.0 repositories
+
+The first step is to remove the Repos 1.0 entry from your machines repository list.
+
+### Debian/Ubuntu
+
+Run
+
+```
+sudo su -c 'rm /etc/apt/sources.list.d/edb-$(lsb_release -cs).list'
+```
+
+
+### RHEL and derivatives
+
+Run
+
+```
+sudo rm /etc/yum.repos.d/edb.repo
+```
+
+
+### SUSE
+
+Run
+
+```
+zypper rr https://zypp.enterprisedb.com/suse/edb-sles.repo
+```
+
+
+## Add Repos 2.0 Repositories
```

Comment:
I think we need to link out to the other 'add the repo web/CLI' docs here because otherwise we retread all the same old ground with 'what about PGD?' etc. Perhaps we need some preamble about how to determine what repos you need?